src/PHPUnit/Constraint/Crawler.php

```diff
@@ -45,26 +56,30 @@ protected function fail($nodes, $selector, ComparisonFailure $comparisonFailure
         }
         $output .= "\ncontains text '{$this->string}'";
 
-        throw new \PHPUnit\Framework\ExpectationFailedException(
+        throw new ExpectationFailedException(
             $output,
             $comparisonFailure
         );
     }
 
-    protected function failureDescription($other) : string
+    /**
+     * @param mixed $other
+     * @return string
+     */
+    protected function failureDescription($other): string
     {
-        $desc = '';
+        $description = '';
         foreach ($other as $o) {
-            $desc .= parent::failureDescription($o->textContent);
+            $description .= parent::failureDescription($o->textContent);
         }
-        return $desc;
+        return $description;
     }
 
-    protected function nodesList(DomCrawler $nodes, $contains = null)
+    protected function nodesList(SymfonyDomCrawler $domCrawler, $contains = null): string
```

Comment:
In the only place where `nodesList` method is used, it is called with second parameter `$this->string` and $contains is only used as string so parameter can have `string` type too.